composer.json

```diff
@@ -9,13 +9,14 @@
         }
     ],
     "require": {
+        "php": "^7.0 || ^8.0",
         "phpcr/phpcr-migrations": "^1.1",
         "phpcr/phpcr-implementation": "^2.1",
         "doctrine/phpcr-bundle": "^1.3 || ^2.0",
-        "symfony/config": "^2.8 || ^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/console": "^2.8 || ^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^2.8 || ^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^2.8 || ^3.4 || ^4.0 || ^5.0 || ^6.0"
+        "symfony/config": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/console": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0 || ^6.0"
```

Comment:
would be fine to drop < 5 i would think

Reply:
I would like to keep `^4.4` as we still support that in Sulu.